packages/block-library/src/list-item/list-style-type.native.js

```diff
@@ -137,3 +145,5 @@ export default function ListStyleType( {
 		/>
 	);
 }
+
+export default withPreferredColorScheme( ListStyleType );
```

Comment:
What do you think if we use the `usePreferredColorScheme` hook instead?

Reply:
Updated! I forgot there was a hook already -- that's much cleaner than the HOC. Thanks for the suggestion. 🤠